controllers/api.controller.js

```diff
@@ -0,0 +1,19 @@
+const endpointsJson = require('../endpoints.json');
+const { selectTopics } = require('../models/api.model');
+
+
+const getApi = (req, res) => {
+    res.status(200).send({ endpoints: endpointsJson });
+};
+
+const getTopics = (req, res, next) => {
```

Comment:
This controller is currently in your api.controller.js file, but it should be in one named topics